src/app/location/Location.js

```diff
@@ -246,7 +246,9 @@ define([
                     // this needs to be here since it sets the text on the verify btn
                     // and it needs to wait for $(that.verifyMapBtn).button('loading'); to finish
                     // which is async
-                    that.addLineToMap(inProgressData.path);
+                    if (inProgressData && inProgressData['path']) { // eslint-disable-line dot-notation
```

Comment:
No need to used []'s here. Missing property will just return undefined.
![image](https://user-images.githubusercontent.com/1326248/36906085-3ef73cac-1df2-11e8-8ffa-97691dd7cad4.png)

Reply:
The issue is that in progress data was null which does throw.

Reply:
Yes, I understand that. I was just suggesting something like this:
```js
if (inProgressData && inProgressData.path) {
...
```

No need for brackets or eslint stuff...